inst/tutorials/r_and_rstudio_basic/r_and_rstudio_basic.Rmd

```diff
@@ -0,0 +1,378 @@
+
+`Saanich_OTU_metadata.csv` is a comma-delimited table of counts of four OTUs in each sample, normalized to 100,000 sequences per sample and the corresponding conditions of each sample (Depth, NO2, NO3 etc.). 
+
+For a brief introduction to these data, see Hallam SJ et al. 2017. Monitoring microbial responses to ocean deoxygenation in a model oxygen minimum zone. Sci Data 4: 170158 [doi:10.1038/sdata.2017.158](https://www.nature.com/articles/sdata2017158).
+
+### Loading tabular data
+
+Data tables can be loaded into R using the tidyverse `read_*` function.
+
+in your file browser, create a `data` directory in your project directory. Download the [`Saanich_OTU_metadata.csv`](https://github.com/EDUCE-UBC/educer/blob/master/data-raw/Saanich_OTU_metadata.csv) file and save it in your `data` directory.
+
+For example, we can load our Saanich data into R with `read_csv` for a comma-separated file and specify the arguments that describe our data as follows:
+
+- `file`: the name of the file you want to load
+- `col_names`: can take either the value `TRUE` or `FALSE` and tells R if the first row contains column names
+
+```{r, eval = FALSE}
+read_csv(file="data/Saanich_OTU_metadata.csv", col_names = TRUE)
+```
+
+```{r, echo = FALSE}
+OTU_metadata_table <- combined
+```
+
+### Save data in the environment
+
+Since we want to do more with our data after reading it in, we need to save it as a variable in R as we did previously with the `<-` operator. You can choose to name the object whatever you like, though this module assumes the names used below.
+
+```{r eval = FALSE}
+OTU_metadata_table <- read_csv(file="data/Saanich_OTU_metadata.csv", col_names = TRUE)
+```
+
+### Data exploration
+
+Let's explore the data that we've imported into R.
+
+Using different functions, we can look at the dimensions of our data, number of rows, and number of columns:
+```{r}
+#number of rows followed by number of columns
+dim(OTU_metadata_table)
+
+#number of rows
+nrow(OTU_metadata_table)
+
+#number of columns
+ncol(OTU_metadata_table)
+```
+
+We can list the column names using `colnames()`:
+```{r}
+colnames(OTU_metadata_table)
+```
+
+We can select to work with only specific columns/variables from our table using the `select()` function:
+```{r}
+restricted_columns <- select(OTU_metadata_table, Depth, OTU0001, OTU0002,
+                             OTU0004)
+```
+
+We can filter for specific rows based on a column value using the `filter()` function. Here we restrict for rows where the value for `Depth` is smaller than 135. 
+
+```{r}
+above_135_depth <- filter(OTU_metadata_table, Depth < 135)
+```
+
+We can also only choose to work with specific rows based on their position in our data table using the `slice()` function. 
+
+```{r}
+first_five_rows <- slice(OTU_metadata_table, 1:5)
+```
+
+
+
+## Getting Help
```

Comment:
Is this not redundant with a section above?

Reply:
Yes, this should be restructured.